### ISSUE TITLE
Fix failing 1.5 Jenkins tests

### ISSUE THREAD

**Issue by suyograo** [elasticsearch/logstash#2718]
http://build-eu-00.elasticsearch.org/job/logstash_regression_15/jdk=JDK7,label=metal-pool/180/consoleFull

```
[plugin:install-development-dependencies] Installing development dependencies of all installed plugins
Installing logstash-devutils
Installation successful
rake aborted!
LoadError: no such file to load -- ci/reporter/rspec
```

**Comment by suyograo** [comment]
@purbon is this related to the new version of devutils?

**Comment by purbon** [comment]
@suyograo it should not be as ```ci/reporter/rspec``` is a dependency only used by jenkins, and installed by logstash-core.gemspec, as you can see https://github.com/elasticsearch/logstash/blob/master/logstash-core.gemspec#L81.

This looks like something happening that provoke development dependencies to be wrong installed, @colinsurprenant @ph  ideas? will investigate this and go back with a fix.

**Comment by purbon** [comment]
Fixed in #2726
